app/[lang]/error.tsx

```diff
@@ -22,28 +22,23 @@ export default function Error({
   }, [error])
 
   return (
-    <div className="flex h-full grow items-end justify-center">
+    <div className="w-full bg-[#2A3B61]">
+      <Topbar />
+
       <Image
-        src="/assets/images/404.jpg"
+        src="/assets/images/error-page_cat.jpg"
         alt="page not found"
-        fill
-        quality={100}
-        loading="eager"
-        priority
-        className="-z-20 h-full w-full object-cover"
+        width={250}
+        height={250}
+        className="absolute top-1/4 left-1/2 z-0 flex -translate-x-1/2 -translate-y-1/2 flex-col justify-center"
       />
-
-      <Topbar />
       <div className="absolute top-1/2 left-1/2 z-0 flex w-screen -translate-x-1/2 -translate-y-1/2 flex-col justify-center">
         <h1 className=" text-center font-cbrush text-6xl text-white">
           {t('error.first')}
         </h1>
         <h1 className="text-center font-cbrush text-4xl text-white">
           {t('error.second')}
         </h1>
-        <h1 className="text-center font-cbrush text-4xl text-white">
-          {t('error.third')}
-        </h1>
 
         <div className=" mt-8 text-center font-cbrush text-2xl">
           <Button href="/">{t('error.back_safety')}</Button>
```

Comment:
Let's make this button have an md:w-auto and w-full so that the button is full width on small screens